tools/gulp/tasks/aot.ts

```diff
@@ -1,20 +1,46 @@
 import {task} from 'gulp';
+import {existsSync} from 'fs';
 import {join} from 'path';
 import {buildConfig, sequenceTask} from 'lib-build-tools';
 import {execTask} from '../util/task_helpers';
 
-const {packagesDir} = buildConfig;
+const {outputDir, packagesDir, projectVersion} = buildConfig;
 
 /** Path to the demo-app source directory. */
+const genericName = 'angular-flex-layout.tgz';
 const demoAppSource = join(packagesDir, 'apps', 'demo-app');
+const tarName = `angular-flex-layout-${projectVersion}.tgz`;
+const distDir = join(outputDir, 'releases', 'flex-layout');
+const genericTar = join(distDir, genericName);
 
 /** Build the demo-app and a release to confirm that the library is AOT-compatible. */
-task('aot:build', sequenceTask('clean', 'flex-layout:build-release', 'aot:run'));
-task('aot:run', sequenceTask('aot:deps', 'aot:cli', 'aot:clean'));
+task('aot:build', sequenceTask('aot:pre', 'aot:cli'));
+
+task('aot:pre', sequenceTask(
+  'clean',
+  'flex-layout:build-release',
+  'aot:bundle',
+  'aot:bundle:rename',
+  'aot:clean',
+  'aot:deps',
+  'aot:add:tar')
+);
 
 task('aot:deps', [], execTask(
   'npm', ['install'], {cwd: demoAppSource}));
 
+task('aot:add:tar', [], execTask(
```

Comment:
Please add comment paragraph here to explain your intent.

Reply:
As cited above, but will add to the file